process-engine/bpmn-studio
# :sparkles: Logs out the user, when changing the backend route

## What did you change?

This PR fixes the bug that occurs when changing the backend while logged in. 
Also it fixes #382 

Fixes #427
Closes #382 

See also https://github.com/process-engine/bpmn-studio/issues/455

## How can others test the changes?

- Checkout the branch & start bpmn-studio
- Login
- Change the backend route via settings
- See that the user is logged out

## PR-Checklist

Please check the boxes in this list after submitting your PR:
